test/sqllogictest/transactions.slt

```diff
@@ -588,3 +588,26 @@ SELECT INTERVAL '1 day'
 
 statement ok
 COMMIT
+
+statement ok
+SET REAL_TIME_RECENCY TO TRUE
```

Comment:
How does this differ from what we currently have?

Reply:
real time recency is set to false by default. The result itself should be identical to when real time recency is turned off. This is mostly to test that things don't blow up when we add the async break in the middle of query execution.